README.md

```diff
@@ -1,15 +1,142 @@
 # Sourceloop Angular Boiler Plate
 
-### Feedback
+<!-- DOCUMENTATION -->
 
-If you've noticed a bug or have a question or have a feature request, [search the issue tracker]([Issues · sourcefuse/angular-boilerplate · GitHub](https://github.com/sourcefuse/angular-boilerplate/issues)) to see if someone else in the community has already created a ticket. If not, go ahead and [make one](https://github.com/sourcefuse/angular-boilerplate/issues/new/choose)! All feature requests are welcome. Implementation time may vary. Feel free to contribute the same, if you can. If you think this extension is useful, please [star](https://help.github.com/en/articles/about-stars) it. Appreciation really helps in keeping this project alive.
+# Description
 
-### Code of Conduct
+This boilerplate project is a project set up that can be easily altered to create new projects. The user is able to use in the original project, its foundation, and its structure to set up a new one without changing the original.
 
-Code of conduct guidelines [here.](CODE_OF_CONDUCT.md)
+This project is based on Angular CLI on version 14.0.0.
 
-### License
+# Prerequisite
 
-[MIT](LICENSE)
+Run `npm install -g @angular/cli` for Angular CLI & NPM installed
+Run `npm i @angular/material` for Angular material
 
-<!-- docs-index-end -->
+# Code scaffolding
+
+Run ng generate component component-name to generate a new component. You can also use ng generate directive|pipe|service|class|guard|interface|enum|module.
+
+## Project Features
+
+Angular Boilerplate is a starter template for Angular applications that provides various features and components to help developers get started with their projects quickly. Here are some of the features and components that the boilerplate provides:
+
+# Auth Module:
+
+This Module provides the following features:
+
+## Login Component:
+
+- login component allows users to authenticate themselves by entering their credentials, such as a username and password. The login component typically includes a form with input fields for the user's credentials, as well as a submit button to initiate the login process.
+
+- login component communicates with a backend server to verify the user's credentials and grant access to the application if the user is authorized. Upon successful authentication, the login component typically stores the user's session information, such as a token or cookie, to allow the user to access protected resources without having to log in again.
```

Comment:
Same with this, these content seems to be the generalised statements. Include specifics of what this boilerplate have.